inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -185,7 +196,35 @@ def from_curves_file(cls,
 
         return cls(coils, num_field_periods)
 
-    def simulate(self):
+    def simulate(
+        self,
+        simulator: Simulator = Simsopt(),
+        resource_pool_id: Optional[UUID] = None,
+        run_async: bool = False,
+        magnetic_field_filename: str = 'magnetic_field.npz',
+        plasma_surface_filename: str = 'input.QA',
+    ) -> tasks.Task:
+        """Simulates the scenario.
+
+        Args:
+            simulator: The simulator to use for the simulation.
+            resource_pool_id: The resource pool to use for the simulation.
```

Comment:
This is missing the description of some args, at least `plasma_surface_filename`.